Lombiq.Tests.UI/Extensions/OrchardCoreDashboardUITestContextExtensions.cs

```diff
@@ -15,26 +16,47 @@ public static void ClickPublish(this UITestContext context, bool withJavaScript
         {
             if (withJavaScript)
             {
-                context.ExecuteScript($"document.querySelector('.publish-button').click()");
+                context.ExecuteScript("document.querySelector('.publish-button, .publish.btn').click();");
             }
             else
             {
                 context.ClickReliablyOn(By.Name("submit.Publish"));
             }
         }
 
-        public static void GoToContentItemList(this UITestContext context)
+        public static void ClickModalOk(this UITestContext context) => context.ClickReliablyOn(By.Id("modalOkButton"));
+
+        /// <summary>
+        /// Sometimes the Publish button doesn't get clicked. This method retries pressing it up to 4 times with a 30
+        /// second interval between attempts. This should grant enough time to execute the publish action if the button
+        /// actually got pressed.
+        /// </summary>
+        public static void ClickPublishReliably(this UITestContext context, bool withJavaScript = false)
```

Comment:
Why would we not use this every time?

Reply:
This will fail if no navigation has occurred. That's not applicable for instance if some client side validation is happening that we expect to fail.

Reply:
I see. Then the name should rather reflect this, because unlike the other *Reliably methods you'd only use this in specific scenarios.

Reply:
Is `ClickPublishUntilNavigation` ok?

Reply:
Yep.